### PR TITLE
NIFI-4952 Add EL support for JettyWebSocketClient and JettyWebSocketS…

### PR DESCRIPTION
**Pull request by claudiu-stanciu**
…erver properties

Thank you for submitting a contribution to Apache NiFi.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [X] Is there a JIRA ticket associated with this PR? Is it referenced 
     in the commit message?

- [X] Does your PR title start with NIFI-XXXX where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.

- [X] Has your PR been rebased against the latest commit within the target branch (typically master)?

- [X] Is your initial contribution a single, squashed commit?

### For code changes:
- [X] Have you ensured that the full suite of tests is executed via mvn -Pcontrib-check clean install at the root nifi folder?
- [ ] Have you written or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)? 
- [ ] If applicable, have you updated the LICENSE file, including the main LICENSE file under nifi-assembly?
- [ ] If applicable, have you updated the NOTICE file, including the main NOTICE file found under nifi-assembly?
- [ ] If adding new Properties, have you added .displayName in addition to .name (programmatic access) for each of the new properties?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

File: nifi-nar-bundles/nifi-websocket-bundle/nifi-websocket-services-jetty/src/main/java/org/apache/nifi/websocket/jetty/JettyWebSocketClient.java [hunk 1]
```diff
@@ -142,10 +142,10 @@ public void startClient(final ConfigurationContext context) throws Exception{
         client.start();
         activeSessions.clear();
 
-        webSocketUri = new URI(context.getProperty(WS_URI).getValue());
-        connectionTimeoutMillis = context.getProperty(CONNECTION_TIMEOUT).asTimePeriod(TimeUnit.MILLISECONDS);
+        webSocketUri = new URI(context.getProperty(WS_URI).evaluateAttributeExpressions().getValue());
```
Comment (MikeThomsen) [inline nifi-nar-bundles/nifi-websocket-bundle/nifi-websocket-services-jetty/src/main/java/org/apache/nifi/websocket/jetty/JettyWebSocketClient.java]: It looks like you're still using `expressionLanguageSupported(boolean)` in this PR. Please rebase off of the latest master and add change it `expressionLanguageSupported(ExpressionLanguageScope.VARIABLE_REGISTRY)`.

**Comment by MikeThomsen** [comment]
@claudiu-stanciu I think this will be ready to merge once you get that change in.

**Comment by MikeThomsen** [comment]
@claudiu-stanciu Looks like someone already made equivalent changes. When I rebased on master to bring it up to 1.7.0, the changes had already been added to master by another commit.

So you can close this ticket. Thanks for the patch.

**Comment by claudiu-stanciu** [comment]
@MikeThomsen sure. Sorry, didn't have time to apply the changes in time.

**Pull request closed without merging**